src/lib/components/ui/Beehiiv.svelte

```
@@ -0,0 +1,41 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
```

ESLint

[error] 'onMount' is defined but never used.

(@typescript-eslint/no-unused-vars)